frontend/app/src/components/StreamlitDialog/StreamlitDialog.tsx

```diff
@@ -110,58 +104,13 @@ interface AboutProps {
 
 /** About Dialog */
 function aboutDialog(props: AboutProps): ReactElement {
-  if (props.aboutSectionMd) {
-    const markdownStyle: CSSProperties = {
-      overflowY: "auto",
-      overflowX: "hidden",
-      maxHeight: "35vh",
```

Comment:
I think we can remove the scrolling/max height here. We also don't have that on other dialogs that can be long (namely `st.dialog`) and I think in reality nobody will write crazy long about messages anyway.